nose_parameterized/parameterized.py

```diff
@@ -211,11 +211,53 @@ def expand(cls, input, testcase_func_name=None):
             >>>
             """
 
-        def parameterized_expand_wrapper(f):
+        def doc_expansion(f, args, kwargs, is_method):
+            offset = 1 if is_method else 0
+            vns = inspect.getargspec(f)[0][offset:]
+            descs = ["{0} = {1}".format(n, repr(v)) for n, v in zip(vns, args)]
+            descs.extend(["{0} = {1}".format(k, repr(v)) for k, v in kwargs.items()])
+
+            # Make up for the shortfall by setting the rest to None
+            descs.extend(["{0} = {1}".format(n, "None") for n in vns[len(args) + len(kwargs.keys()):]])
```

Comment:
This is incorrect when the default value is not `None` (ex, `def foo(self, bar=42)`).